tree_test.go

```diff
@@ -221,8 +232,8 @@ func TestTreeMoar(t *testing.T) {
 		{m: mGET, r: "/articles/456/data.json", h: hStub11, k: []string{"id"}, v: []string{"456"}},
 
 		{m: mGET, r: "/articles/files/file.zip", h: hStub12, k: []string{"file", "ext"}, v: []string{"file", "zip"}},
-		{m: mGET, r: "/articles/files/photos.tar.gz", h: hStub12, k: []string{"file", "ext"}, v: []string{"photos", "tar.gz"}},
-		{m: mGET, r: "/articles/files/photos.tar.gz", h: hStub12, k: []string{"file", "ext"}, v: []string{"photos", "tar.gz"}},
+		{m: mGET, r: "/articles/files/photos.tar.gz", h: hStub12, k: []string{"file", "ext"}, v: []string{"photos.tar", "gz"}},
```

Comment:
I would call it breaking :/

Reply:
Maybe add a test case that parses this correctly? It should be called out in the changelog how to fix such cases.

Reply:
Sorry, I dont understand what you mean / what I have to do

Reply:
May have been a brain fart. I do see you added positive test cases above.

Not sure how to proceed here. These cases do look bad. I think a option whether parameters catch a dot may be needed, ideally on route level.